src/lib/config/common.js

```diff
@@ -8,10 +8,10 @@ export const DEFAULT_LANGUAGE = 'ENG';
 // text key is used for semantic ui dropdown options display
 
 export const ACQ_ORDER_STATUSES = [
-  { value: 'CANCELLED', text: 'Cancelled', label: 'Cancelled' },
-  { value: 'RECEIVED', text: 'Received', label: 'Received' },
-  { value: 'ORDERED', text: 'Ordered', label: 'Ordered' },
-  { value: 'PENDING', text: 'Pending', label: 'Pending' },
+  { value: 'PENDING', text: 'Pending', label: 'Pending', order: 1 },
```

Comment:
This is nice, but then you need to add the code that when rendering these statuses, you will have to sort by `order`.
Please add this below in the other statuses too.